.circleci/config.yml

```diff
@@ -187,12 +265,12 @@ jobs:
       - run:
           name: install emsdk
           command: |
-            wget -O ~/emsdk-master.tar.gz https://github.com/juj/emsdk/archive/master.tar.gz
+            curl -# -L -o ~/emsdk-master.tar.gz https://github.com/emscripten-core/emsdk/archive/master.tar.gz
```

Comment:
why curl?

Reply:
Because the macOS didn't come with wget preinstalled.  I guess I could have installed it with brew but this seemed easier.